src/ontology/mondo-edit.obo

```diff
@@ -464825,6 +464835,487 @@ name: neurological muscular channelopathy due to a genetic ryanodine receptor de
+is_a: MONDO:0700092 {source="https://orcid.org/0000-0001-5208-3432"} ! neurodevelopmental disorder
+
+[Term]
+id: MONDO:0957268
+name: hypersulfaturia
+xref: OMIM:620372 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 {source="OMIM:620372"} ! hereditary disease
+
+[Term]
+id: MONDO:0957270
+name: muscular dystrophy, limb-girdle, autosomal recessive 28
+xref: OMIM:620375 {source="MONDO:equivalentTo"}
+is_a: MONDO:0015152 {source="OMIM:620375"} ! autosomal recessive limb-girdle muscular dystrophy
+
+[Term]
+id: MONDO:0957271
+name: autoinflammatory disease, systemic, with vasculitis
+xref: OMIM:620376 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 {source="OMIM:620376", source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
+is_a: MONDO:0019751 {source="https://orcid.org/0000-0001-5208-3432"} ! autoinflammatory syndrome
+
+[Term]
+id: MONDO:0957273
+name: Charcot-Marie-Tooth disease, dominant intermediate A
+xref: OMIM:620378 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 {source="OMIM:620378"} ! hereditary disease
+
+[Term]
+id: MONDO:0957274
+name: spastic paraplegia 89, autosomal recessive
+xref: OMIM:620379 {source="MONDO:equivalentTo"}
+is_a: MONDO:0019064 {source="OMIM:620379"} ! hereditary spastic paraplegia
+
+[Term]
+id: MONDO:0957278
+name: oocyte/zygote/embryo maturation arrest 20
+xref: OMIM:620383 {source="MONDO:equivalentTo"}
+is_a: MONDO:0014769 {source="OMIM:620383"} ! inherited oocyte maturation defect
+
+[Term]
+id: MONDO:0957279
+name: auditory neuropathy, autosomal dominant 2
+xref: OMIM:620384 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 {source="OMIM:620384"} ! hereditary disease
+
+[Term]
+id: MONDO:0957281
+name: nemaline myopathy 5B, autosomal recessive, childhood-onset
+xref: OMIM:620386 {source="MONDO:equivalentTo"}
+is_a: MONDO:0018958 {source="OMIM:620386"} ! nemaline myopathy
+
+[Term]
+id: MONDO:0957284
+name: nemaline myopathy 5C, autosomal dominant
+xref: OMIM:620389 {source="MONDO:equivalentTo"}
+is_a: MONDO:0018958 {source="OMIM:620389"} ! nemaline myopathy
+
+[Term]
+id: MONDO:0957288
+name: intellectual developmental disorder, autosomal recessive 79
+xref: OMIM:620393 {source="MONDO:equivalentTo"}
+is_a: MONDO:0019502 {source="OMIM:620393"} ! autosomal recessive non-syndromic intellectual disability
+
+[Term]
+id: MONDO:0957294
+name: pulmonary fibrosis and/or bone marrow failure syndrome, telomere-related, 9
+xref: OMIM:620400 {source="MONDO:equivalentTo"}
+is_a: MONDO:0000148 {source="OMIM:620400"} ! pulmonary fibrosis and/or bone marrow failure, telomere-related
+
+[Term]
+id: MONDO:0957301
+name: spermatogenic failure 84
+xref: OMIM:620409 {source="MONDO:equivalentTo"}
+is_a: MONDO:0004983 {source="OMIM:620409"} ! spermatogenic failure
+
+[Term]
+id: MONDO:0957302
+name: low density lipoprotein cholesterol level quantitative trait locus 3
+xref: OMIM:620410 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 {source="OMIM:620410"} ! hereditary disease
```

Comment:
it looks like this term should be excluded as it is not a disease.

Reply:
good catch, thanks